src/coreComponents/physicsSolvers/PhysicsSolverBase.cpp

```diff
@@ -760,6 +760,29 @@ real64 PhysicsSolverBase::eisenstatWalker( real64 const newNewtonNorm,
   return krylovTol;
 }
 
+
+void MemoryUsageOutput( std::string const & message )
```

Comment:
was this for debugging? Or do you need it? In any case I don't think it should be in `PhysicsSolverBase`

Reply:
This is for debugging, I'll update you guys soon